dll/win32/shell32/lang/bg-BG.rc

```diff
@@ -322,11 +322,11 @@ BEGIN
     EDITTEXT 14007, 110, 103, 86, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     EDITTEXT 14008, 197, 103, 33, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     CONTROL "", 14015, "Static", SS_OWNERDRAW, 70, 116, 100, 40
-    CTEXT "Устройство %c", 14009, 95, 162, 50, 10
-    PUSHBUTTON "Почистване на &диска", 14010, 160, 159, 70, 15, WS_TABSTOP
+    CTEXT "диск %c", 14009, 95, 162, 50, 10
```

Comment:
Uncapitalized?

Reply:
It's done like that everywhere and always was like that. I have never questioned that. It works(TM). And all languages do it like that.